pcf-with-installed-tile/bin/0-check-dependencies

```diff
@@ -22,12 +22,6 @@ fi
 
 cf target -o cyberark-conjur-org -s cyberark-conjur-space
 sb_name=$(cf apps | grep conjur-service-broker | awk '{print $1;}')
-version=$(cf env $sb_name | grep CONJUR_VERSION | awk '{print $2;}')
-
-if [[ "$version" -ne "5" ]]
-then
-  fatal "Your service broker must have CONJUR_VERSION set to 5 to run this demo."
```

Comment:
why would we delete this? based on my reading of it, it seems like it'd be useful to verify that the version is 5

Reply:
at the least, we should also have a note in the README in this dir to explain that you need to use Conjur v5.

Reply:
Setting this env var is just extra work when the demo only runs with one version. I added a note to the top-level README but yeah it's probably worth adding to the subdir ones as well.